server/routes/journeys/dailySchedule/handlers/downloadCsvHandler.ts

```diff
@@ -26,9 +27,28 @@ export default class DownloadCsvHandler implements PageHandler {
       user,
     )
 
-    const csv = converter.json2csv(schedule.appointmentGroups.flat())
+    const csv = converter.json2csv(this.convertScheduleToCsvRows(schedule))
     res.header('Content-Type', 'text/csv')
     res.attachment(`daily-schedule${status === 'CANCELLED' ? '-cancelled' : ''}-${formatDate(date, 'yyyy-MM-dd')}.csv`)
     res.send(csv)
   }
+
+  private convertScheduleToCsvRows = (schedule: DailySchedule) => {
+    return schedule.appointmentGroups.flatMap(group =>
+      group.map(a => ({
+        prisonerName: convertToTitleCase(`${a.prisoner.firstName} ${a.prisoner.lastName}`),
+        prisonerNumber: a.prisoner.prisonerNumber,
+        cellNumber: a.prisoner.cellLocation,
+        appointmentStartTime: a.startTime,
+        appointmentEndTime: a.endTime,
+        appointmentType: a.appointmentDescription,
```

Comment:
`endTime` could be null for NOMIS appointments. Possible a null-check and replace with empty string?

Reply:
Should we even be displaying appointments without an end time?